### ISSUE TITLE
Accessibility violations in common properties tables and toggletext

### ISSUE THREAD

**Issue by caritaou** [elyra-ai/canvas#2490]
ref: 4046

![Image](https://github.com/user-attachments/assets/fa119e19-6751-4b20-a043-6da15980f98d)

![Image](https://github.com/user-attachments/assets/79a2e634-61ac-4dcc-87ad-7e7b8392707f)
